collectors/statsd.plugin/asterisk.md

```diff
@@ -0,0 +1,61 @@
+<!--
+title: "Asterisk monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/statsd.plugin/asterisk.md
+
+sidebar_label: "Asterisk"
+-->
+
+# Asterisk monitoring with Netdata
+
+Monitors [Asterisk](https://www.asterisk.org/) dialplan application's statistics.
+
+## Requirements
+
+- Asterisk [integrated with StatsD](https://www.asterisk.org/integrating-asterisk-with-statsd/).
+
+## Configuration
+
+Netdata ships
+with [asterisk.conf](https://github.com/netdata/netdata/blob/master/collectors/statsd.plugin/asterisk.conf) with
+preconfigured charts.
+
+You only need to configure Asterisk to send statistics to Netdata. The following suffices for `/etc/asterisk/statsd.conf`:
+
+```ini
+[general]
+enabled = yes                   ; When set to yes, statsd support is enabled
+server = 127.0.0.1              ; server[:port] of statsd server to use.
+                                ; If not specified, the port is 8125
+prefix = asterisk               ; Prefix to prepend to all metrics
+```
+
```

Comment:
I would like to include a sentence that describes the result. Does the Asterisk application open a server on 127.0.0.1:8125 and Netdata listens on that server? If yes, how does Netdata recognize that?

Reply:
Netdata is a StatsD server that listens on 127.0.0.1:8125. Asterisk is a client.

```cmd
ilyam@deb-fra1-01:~$ sudo ss -ntlp | grep netdata
[sudo] password for ilyam:
LISTEN 0      4096       127.0.0.1:8125       0.0.0.0:*    users:(("netdata",pid=171508,fd=28))
LISTEN 0      4096         0.0.0.0:19999      0.0.0.0:*    users:(("netdata",pid=171508,fd=4))
LISTEN 0      4096            [::]:19999         [::]:*    users:(("netdata",pid=171508,fd=5))
```